training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,716 @@
+{
```

Comment:
<p><code>separate EC2 instance/notebook</code>  -&gt; Clarify <code>separate EC2 instance or SageMaker notebook</code>  ?</p><p>Typo - <code>amazon-sagemaker-examples</code>  repository (missing plural "s" on examples)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Choosing "a SageMaker notebook instance"

Reply:
<p>Also I mentioned "continuous integration (CI) test" in this markdown cell.</p>